samples/nrf_rpc/protocols_serialization/server/Kconfig

```diff
@@ -22,6 +22,20 @@ config LOG_BACKEND_RTT
 config LOG_BACKEND_UART
 	default n
 
+if SOC_NRF54L15
+
+config BT_CTLR_ECDH
+	default n
+
+config BT_USE_PSA_API
+	default y if !BT_CTLR_ECDH
```

Comment:
Why do not use PSA if ECDH was enabled? PSA has ECDH implementation.

Reply:
That sounds reasonable. However, I doubt in that Host will use Cracen driver. It uses mbedtls. Will mbedtls use Cracen driver as backend?

Reply:
Will ble host and ot use the same PSA implementations?

Reply:
I see discussion in NCS Bluetooth channel. Let keep what you suggested for now.